specification/communication/data-plane/readme.md

```diff
@@ -84,6 +84,9 @@ These settings apply only when `--tag=package-2021-03-07` is specified on the co
 ```yaml $(tag) == 'package-2021-03-07'
 input-file:
   - Microsoft.CommunicationServicesIdentity/stable/2021-03-07/CommunicationIdentity.json
```

Comment:
I realized this is called `CommunicationIdentity` should this be aligning with the naming used here to `CommunicationIdentifier`?

@RezaJooyandeh @DominikMe

Reply:
That is for the identity service. I think that should stay as is. We can think about in terms of that service would manage all identities and identifiers are their address